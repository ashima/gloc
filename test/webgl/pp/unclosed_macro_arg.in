#define F(A) (A)
void main() {
     F ( 5
}
