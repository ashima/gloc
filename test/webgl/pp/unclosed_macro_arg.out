File 0, line 3, col 7 - 8:
unclosed macro argument list

Fatal: unrecoverable preprocessor error (5)
